src/follower.py

```diff
@@ -73,35 +73,60 @@ def stop(self):
 
     """
     Turn the wheels towards the last recognized object.
+    Pans the camera towards the last recognized object.
     """
     def turn(self):
+        self.pan_camera()
         self.convert_camera_angle()
         self._fw.turn(self._turn_angle)
 
 
     """
-    Converts the camera's angle scale to the same scale as the wheels.
+    Follows the tag by panning camera towards the same direction as the wheels.
+    
+    If the direction of the object is more/less than the distance then set the camera 3 steps 
+    to the left or right. Otherwise reset the camera to 90 degrees or 0 steps. 
+    """
+    def pan_camera(self):
+        if self._turn_angle < self._distance:
+            self.turn_camera_left(3)
+        elif self._turn_angle > self._distance:
+            self.turn_camera_right(3)
+        else:
+            self.reset_camera() 
+
+
+    """
+    Definitions for turning the camera left and right.
+    Parameter: Steps 
+    """
+    def turn_camera_left(self, step):
+        self._camera.turn_left(step)
+    
+    def turn_camera_right(self, step):
+        self._camera.turn_right(step)
+
 
+    """
+    Converts the camera's angle scale to the same scale as the wheels.
     The camera reports objects directly in front of it as 90 degrees. Everything
     to the left of center is negative ranging from [-45, -90) with -45 being the
     leftmost angle. Everything to the right of center is positive ranging from
     (90, 135] with 135 being the rightmost angle.
-
     The wheels turn on a range of [45, 135] with 45 being the rightmost, 135
     being the leftmost, and 90 being center.
     """
     def convert_camera_angle(self):
         if self._decision == -1:
-            self._turn_angle *= -1
+            self._turn_angle = np.abs(self._yaw - 90)
```

Comment:
Same as earlier, this is already merged and in the repository. Remove these changes and pull the changes from master.